cloud/deploys/bin/azure/db-connection

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env bash
+
+# DOC: Connect to the database via the bastion
+# example: ./db-connection -g sgdev -d civiform-artistic-halibut
+key_name="bastion"
+while getopts g:d:k flag
+do
+    case "${flag}" in
+        g) resource_group=${OPTARG};;
+        d) database_name=${OPTARG};;
+        k) key_name=${OPTARG};;
+    esac
+done
+
+if [[ ! -z "${resource_group}" ]] || [[ ! -z "${database_name}" ]]; then
+  echo "arguments -g and -d must be provided"
+  exit 1
+fi
+
+key_file="$HOME/.ssh/${key_name}"
+if ! [[ -f "${key_file}" ]]; then
+    echo "Creating a new ssh key"
+    ssh-keygen -t rsa -b 4096 -f "${key_file}"
+fi
+
+echo "Getting the bastion vm's ip address"
+vm_ip_address=$(az network public-ip show \
+    -g "${resource_group}" \
+    -n "${resource_group}-ip" \
+    --query "ipAddress" | tr -d '"')
+
+echo "Getting the postgres host name"
+postgres_host=$(az postgres server show \
+    -g "${resource_group}" \
+    -n "${database_name}" \
+    --query "fullyQualifiedDomainName" | tr -d '"')
+
+# echo "Adding the public key to the bastion vm"
+az vm user update \
+    -u adminuser \
+    -g "${resource_group}" \
+    -n "${resource_group}-bstn-vm" \
+    --ssh-key-value "$(< ~/.ssh/${key_name}.pub)"
```

Comment:
question (blocking): how can we clean these up after access is finished so we don't have permanent ingress to the DB?

Reply:
well i guess is this something we need to do?

Reply:
yes

Reply:
cool okay i remove it from the users machine